NIAD-272 Introduce a polling cycle duration

mattd-kainos opened the pull request:
…" to prevent multiple instances from downloading

mattd-kainos committed:
NIAD-272 Polling cycle stops if it runs for more than the "scan delay" to prevent multiple instances from downloading

mattd-kainos committed:
Renamed mesh timing variables

mattd-kainos committed:
app yaml update

mattd-kainos committed:
README update

mattd-kainos committed:
variable rename

grzegorz-robak commented on src/main/java/uk/nhs/digital/nhsconnect/nhais/mesh/MeshService.java:
```diff
@@ -22,57 +23,82 @@
                 }
             }
-            LOGGER.info("Completed MESH mailbox polling cycle");
+            LOGGER.info("Completed MESH mailbox polling cycle. Processed all messages from inbox.");
```
Wouldn't this line execute even when not all messages from inbox were processed?

mattd-kainos replied:
Correct - I have changed the `break` to a `return` so that this is only execute if the loop completes w/o short-circuit.

mattd-kainos committed:
readme dockerfile jmeter

mattd-kainos committed:
Changelog
docker-compose default for responder
reduced an amendment controller log to info

mattd-kainos committed:
change short-circuit from break to return
